### DISCUSSION TITLE
keybind for toggling side pannels

### DISCUSSION THREAD

**Discussion by lmcr666** [Floorp-Projects/Floorp#1422] [Feature Request]
I have whatsapp web on a side pannel and wanted a keybind to toggle it so i dont have to use my mouse to click it.
i've read through [this thread and even left my own comment on it ](https://github.com/Floorp-Projects/Floorp/discussions/556#discussioncomment-10509124)to try and figure out how to do it through custom actions but i got no answer

thank you in advance

**Comment by CutterKnife** [comment]
There should be a Browser Manager Sidebar section in the Keyboard Shortcut settings.

**Reply by lmcr666** [reply]
wow, found it, thank you!!

**Comment by lmcr666** [comment]
for future reference or anyone looking for the same thing
![image](https://github.com/user-attachments/assets/0bbdf14d-401e-47f5-95b3-57d5bf9994eb)
